packages/ui/src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,64 @@
+import { forwardRef } from "react";
+
+import {
+  Arrow,
+  Content,
+  Provider,
+  Root,
+  Trigger,
+} from "@radix-ui/react-tooltip";
+import { twMerge } from "tailwind-merge";
+
+export type TooltipProviderProps = {
+  children: React.ReactNode;
+  /**
+   * The duration from when the pointer enters the trigger until the tooltip gets opened.
+   * @defaultValue 700
+   */
+  delayDuration?: number;
+  /**
+   * How much time a user has to enter another trigger without incurring a delay again.
+   * @defaultValue 300
+   */
+  skipDelayDuration?: number;
+  /**
+   * When `true`, trying to hover the content will result in the tooltip closing as the pointer leaves the trigger.
+   * @defaultValue false
+   */
+  disableHoverableContent?: boolean;
+};
+const TooltipProvider: React.FC<TooltipProviderProps> = Provider;
+
+const Tooltip = Root;
+
+const TooltipArrow = Arrow;
+
+const TooltipTrigger = Trigger;
+
+const TooltipContent = forwardRef<
+  React.ElementRef<typeof Content>,
+  React.ComponentPropsWithoutRef<typeof Content>
+>(({ className, sideOffset = 4, ...props }, ref) => (
+  <Content
+    ref={ref}
+    sideOffset={sideOffset}
+    className={twMerge(
+      "z-50 overflow-hidden rounded-12 p-3 text-sm shadow-3",
+      "text-text-med-em bg-outline-base-em",
```

Comment:
background color should be `bg-surface-special-bg-neutral`, `outline-base-em` is for the border. Let's add a 1px border with the `outline-base-em` color.

Reply:
Thanks! Updated.